Add schema, validation, and better testing (#117)
AlexsLemonade/OpenScPCA-nf: readme-internal.md

#### Using the Nextflow-workload instance

All runs are launched on the `Nextflow-workload` instance in the OpenScPCA AWS workloads account.
You can log in to this instance to monitor _and cancel_ the workflow runs using the AWS console.

> You will first need to have access to the "workload" account, then you can log into the instance using the "Connect" button on the EC2 console.

Reviewer: ```suggestion
You will first need to have access to the "workloads" account, then use the AWS console to navigate to the EC2 console and log into the instance via the "Connect" button. 
```
Minor, but I think its "workloads" plural. Also updating the order here just for clarity.
